src/client/sandbox/event/simulator.js

```diff
@@ -529,6 +529,9 @@ export default class EventSimulator {
     }
 
     _dispatchMouseRelatedEvents (el, args, dataTransfer) {
+        if (shouldIgnoreMouseEventInsideIframe(el, args.clientX, args.clientY) && args.type !== 'mouseover' && args.type !== 'mouseenter')
```

Comment:
I think that the type conditions should be the first for performance improvement.